src/main/java/banking/configuration/DBManager.java

```diff
@@ -0,0 +1,43 @@
+package banking.configuration;
+
+import org.sqlite.SQLiteDataSource;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class DBManager {
+
+    private final static String CREATE_TABLE = """
```

Comment:
The order is important, please use `private static final` one
Please use this resource to revise the Java Code/Naming Conventions: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
fixed